core/indexer/common.go

```diff
@@ -254,12 +270,18 @@ func serializeBulkMiniBlocks(hdrShardID uint32, bulkMbs []*Miniblock) bytes.Buff
 					"mb hash", mb.Hash)
 				continue
 			}
-
 		} else {
 			// update miniblock
 			meta = []byte(fmt.Sprintf(`{ "update" : { "_id" : "%s", "_type" : "%s"  } }%s`, mb.Hash, "_doc", "\n"))
-			serializedData = []byte(fmt.Sprintf(`{ "doc" : { "receiverBlockHash" : "%s" } }`, mb.ReceiverBlockHash))
+			if hdrShardID == mb.SenderShardID {
+				// update sender block hash
+				serializedData = []byte(fmt.Sprintf(`{ "doc" : { "senderBlockHash" : "%s" } }`, mb.SenderBlockHash))
+			} else {
```

Comment:
Can we remove the else here so we'd have less indentation?
```go
serializedData = []byte(fmt.Sprintf(`{ "doc" : { "receiverBlockHash" : "%s" } }`, mb.ReceiverBlockHash))
if hdrShardID == mb.SenderShardID {
    // update sender block hash
    serializedData = []byte(fmt.Sprintf(`{ "doc" : { "senderBlockHash" : "%s" } }`, mb.SenderBlockHash))
}
```

Reply:
maybe we can remove the `else` from this `if` but if we do that we will have two assignments in variable `serializedData`  and one of them is pointless